### ISSUE TITLE
Install feature

### ISSUE THREAD

**Issue by vinicius94** [Rall3n/galaxy-integration-amazon#24]
I checked the history log and saw that you removed the install feature. Why?

I decompiled the applications and know the "right way" calling install doesn't work, but, calling play does the trick. There's a special reason? Have you tried contacting Amazon?

**Comment by Rall3n** [comment]
Hello @vinicius94,

I removed it because it does not work. Calling the `play` scheme command does not do the trick. Here is a screen capture of what happens when you try to use `amazon-games://play/...` to start a game not installed on your computer:

https://user-images.githubusercontent.com/15950630/148259824-ea8038d2-87ab-4877-a59a-223667675467.mp4

Calling the command causes a prompt to appear. Accepting the prompt shows an indication of progress and work, but nothing happens.

> Have you tried contacting Amazon?

I tried, but I could not reach a qualified person responsible for the app.
